src/controllers/account.ts

```diff
@@ -79,12 +75,12 @@ export async function deleteYourProofOfIdPost(
     const kvbRDF = await kvbRDFUri(session);
     const kvbBlob = await getDatasetUri(
       session,
-      "private/govuk/identity/poc/credentials/vcs/kbv/check"
+      "getCheckStoragePath()kbv/check"
     );
     const passportRDF = await passportRDFUri(session);
     const passportBlob = await getDatasetUri(
       session,
-      "private/govuk/identity/poc/credentials/vcs/passport/check"
+      "getCheckStoragePath()passport/check"
```

Comment:
```suggestion
      `${getCheckStoragePath()}/passport/check`
```
You're missing the string interpolation syntax and trailing slash here (and on line 78)

Reply:
urgh gotta stop doing find and replace, juts rebasing and will put this up as a proper PR.